.gitignore

```diff
@@ -31,6 +31,9 @@ util/bot/sde-win32.tar.xz
 util/bot/win_toolchain.json
 util/bot/yasm-win32.exe
 
-test_build_dir
+test_build_dir/
+cmake-build-debug/
+symbols.txt
+/.idea
```

Comment:
Nit: keep these in the global ignore `~/.gitignore`?

Reply:
I'll move`.idea` to my global `~/.gitignore`.  The other 3 lines are very specific to the artifacts created by this projects CI build scripts. Any objections to keeping those 3 lines?